### ISSUE TITLE
ViewPager: Reads not visible views.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#139]
```
ViewPager [1] will preload the previous and next view for fast switching and 
TalkBack reads them, event though they are not shown.

[1] 
http://developer.android.com/reference/android/support/v4/view/ViewPager.html
```

Original issue reported on code.google.com by `reiter.christoph@gmail.com` on 24 Sep 2012 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
Can you provide a specific repro case in an existing application?
```

Original comment by `caseybur...@google.com` on 15 Oct 2012 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
http://pastie.org/5066362
```

Original comment by `reiter.christoph@gmail.com` on 16 Oct 2012 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
Please attempt to reproduce this again with the latest TalkBack v3.2.1_b1.  
We've fixed a bug that failed to check for visibility within focusability tests.
```

Original comment by `caseybur...@google.com` on 20 Nov 2012 at 7:27

**Comment by GoogleCodeExporter** [comment]
```
Assuming this was fixed in TalkBack v3.2.1.
```

Original comment by `alanv@google.com` on 7 Dec 2012 at 6:28
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I checked it on Talkback3.2.1 release apk, this is still not fixed.
```

Original comment by `ivar.i...@gmail.com` on 7 Jan 2013 at 2:10
